.env.template

```diff
@@ -1,4 +1,5 @@
 BASEURL="url"
+BASEURLAPI="url/api"
 EMAIL="email"
 PASSWORD="password"
 USERID="number"
```

Comment:
All variables should be in format:
BASE_URL
BASE_URL_API

So it is more visible